### PR TITLE
maker 2.0 updates

### PR DESCRIPTION
**Pull request by cloudonshore**

### CODE REVIEW

File: docs/tutorials/maker2.0/app.js [hunk 1]
```diff
@@ -29,7 +35,7 @@ const router = new Router(routerParams)
 function priceTrade(params) {
   // Assume a fixed price of 0.01 DAI/WETH
   // You should implement your own pricing logic here.
-  const price = 0.01
+  const price = 200
```
Comment (dmosites) [inline docs/tutorials/maker2.0/app.js]: Either update value or comment above

File: docs/tutorials/maker2.0/app.js [hunk 2]
```diff
@@ -47,7 +53,8 @@ function priceTrade(params) {
     const makerParamDecimals = takerParamDecimals / price
     makerParam = TokenMetadata.formatAtomicValueByToken({address: params.makerToken}, makerParamDecimals)
   }
-
+  console.log('PRICED TRADE', makerParam,
```
Comment (dmosites) [inline docs/tutorials/maker2.0/app.js]: Please remove unless intentional.

**Comment by perich** [comment]
We should check token approval and warn in the console if the token isn't approved for trade. e.g. `"WARNING: all trades will fail until you approve ${token} for trade.`

**Comment by cloudonshore** [comment]
> We should check token approval and warn in the console if the token isn't approved for trade. e.g. `"WARNING: all trades will fail until you approve ${token} for trade.`

@perich  Agreed, will add in a future iteration. 

This is currently step 2 of 7. I was planning to make it more fully fleshed out in step 5. 
```
1. get swapVersion added to the indexer
2. implement a maker example in developers with the new swap functions in airswap.js (minimalistic, just for serving orders to instant.development)
3. implement multiplexing in protocolMessaging for instant checkout paths where the bestOrder is a swap 2.0 order
4. release the multiplexed instant frontend to production (orders can now be served against swap 2.0 in production in parallel to swap 1.0 orders)
5. now we can rework the documentation to be that of a swap 2.0 maker, we release that, and simultaneously:
6. release a blog post describing the changes, pointing to the documentation, and the reference maker, and giving a date that the migration needs to be completed by
7. after a few weeks, we remove the multiplexing and disallow swap 1.0 intents on instant (edited) 
```

**Pull request merged**